### PR TITLE
Merge EyoWeiChin-UGDG-Updates

### LINKED ISSUE #247: [PE-D] Lack of visuals for CS2113T-F12-2
Although there are some excepted output but I felt some visuals e.g., screenshots can be integrated to the explanation, to make the UG more reader-friendly.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: LIU-YiFeng-1/ped#1

### LINKED ISSUE #248: [PE-D] Misleading commands
I was trying out the command watchlist after the `help` command. The app prompt me with the available list of commands but the `watchlist -l` looks very misleading and I typed `watchlist -1` a few time before realizing it. Is it possible to change it to something less misleading? Please refer to the attachment for the screenshot.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: LIU-YiFeng-1/ped#2

### PR DESCRIPTION
**Pull request by EyoWeiChin**
Update
- UG's Using this guide and Quick Start component
- DG's Browse to improve formatting

Fixes #247, #248

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -686,7 +709,7 @@ works. However, we would strongly recommend you not to as you may cause data cor
 | ---                                          | ---                                                      |
 | Help                                         | `help`                                                   |
 | Estimate time needed to translate the script | `estimate <SCRIPT_FILE_NAME> [-wph WORDS_PER_HOUR]`      |
-| Browse                                       | `browse [-s SORT_OPTION] [-p PAGE_NO.] [-o SORT_ORDER]`  |
+| Browse                                       | `browse [-s <SORT_OPTION>] [-p <PAGE_NO.>] [-o <SORT_ORDER>]`  |
```
Comment (OngDeZhi) [inline docs/UserGuide.md]: I think you don't need a <> inside the optional argument bracket, similarly for line 726.

**Review by OngDeZhi — COMMENTED** [review]
Beside the one I mentioned, the rest looks good. Thanks for updating the developer guide and user guide.

File: docs/UserGuide.md (hunk 1 above)
Comment (EyoWeiChin) [reply docs/UserGuide.md]: Thank you for pointing this out, I have rectified the issue, in the latest commit.

**Review by OngDeZhi — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -57,12 +57,31 @@ This guide provides you with in-depth information on setting up AniChan and how
...
+
+Along the way you might encounter several icons. These icons can help provide several types of information that may be useful to you. 
+
+> :bulb: Take note when you see this icon, as it might tell you something important.
+
+> :memo: This icon represents a friendly tip that might be useful when using our application.
+
+> :warning: A caution that you have to keep in mind when using the command
+
+Lastly, text that are blue like this [example](#12-using-this-guide), are clickable links that will bring you to the relevant part of this user guide.
```
Comment (ChanJianHao) [inline docs/UserGuide.md:70]: Thanks for adding the guide!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -57,12 +57,31 @@ This guide provides you with in-depth information on setting up AniChan and how
...
+<br/>
+
 ## 2. Quick Start
 1.  Ensure that you have Java 11 or above installed.
 2.  Download the latest version of **AniChan** from [here](https://github.com/AY2021S1-CS2113T-F12-2/tp/releases/tag/V2.0).
 3.  Copy the file to the folder you want to use as the home folder for **AniChan**.
 4.  Open **command prompt** or **terminal** and change directory into the folder. Run `java -jar AniChan.jar`.
-5.  Type a command into the command prompt and press `Enter` to execute it. For example, typing `help` and pressing `Enter` will display the help message.
+5.  You will be greeted by a prompt for your name and gender as shown below.
+6.  Upon completing these 2 prompts, you can begin using **AniChan**!
```
Comment (ChanJianHao) [inline docs/UserGuide.md:80]: Could it be confusing to mention 2 prompts when line 79 states `a prompt`?

**Review by ChanJianHao — APPROVED** [review]
LGTM :)

File: docs/UserGuide.md (hunk 3 above)
Comment (EyoWeiChin) [reply docs/UserGuide.md:80]: Hmm, you are right, I will amend this in the next PR. Thank you!

**Pull request merged**